internal/venice-test-common/build.gradle

```diff
@@ -312,9 +312,6 @@ task generateGHCI() {
   appendToGHCI(paramFileContent, targetFilePath, otherTest, 120, integTestGradleArgs + "9999")
   jobs << otherTest
 
-  // define a job that depends others to manage the status check
-  appendToGHCI(paramFileContent, targetFilePath, "E2ETestsCompletion", 120, "clean", jobs)
-
```

Comment:
Can we keep this, or change it somehow. It makes creating and maintaining branch protection rules much simpler

Reply:
Sure. I could create a separate generator file for completion.  Currently there is no point to upload artifacts for it anyways.